### PR TITLE
(DOCSP-11307): type check for sync client guides

### PR DESCRIPTION
**Pull request by mongomoe**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-11307

### Docs staging link (requires sign-in on MongoDB Corp SSO):
- android: https://docs-mongodbcom-staging.corp.mongodb.com/realm/mohammad.hunan/DOCSP-11307-type-check-for-sync-client-guides/android/objects.html
- ios: https://docs-mongodbcom-staging.corp.mongodb.com/realm/mohammad.hunan/DOCSP-11307-type-check-for-sync-client-guides/ios/objects.html
- node: https://docs-mongodbcom-staging.corp.mongodb.com/realm/mohammad.hunan/DOCSP-11307-type-check-for-sync-client-guides/node/objects.html

doc w/ summary of types allowed/not allowed:
https://docs.google.com/document/d/1vSPniERWWe-CLh8vcAb9YMjKBozI9rXlzfzPFxXBi2k/edit?usp=sharing

### CODE REVIEW

File: source/node/include/property-schema.rst [hunk 1]
```diff
@@ -22,7 +22,7 @@ You can configure the following constraints for a given property:
        {+client-database+} supports the following primitive data types:
 
        - ``bool`` for boolean values
-       - ``int``, ``float``, and ``double``, which map to JavaScript ``number`` values
+       - ``int``, and ``double``, which map to JavaScript ``number`` values
```
Comment (ccho-mongodb) [inline source/node/include/property-schema.rst:25]: Nitpick: Since this is no longer a list with more than two items, I think you can remove the comma.

**Review by ccho-mongodb — APPROVED** [review]
LGTM, one nitpick.

File: source/node/include/property-schema.rst (hunk 1 above)
Comment (mongomoe) [reply source/node/include/property-schema.rst:25]: 👍

Comment (mongomoe) [inline source/node/include/property-schema.rst:25]:
```suggestion
       - ``int`` and ``double``, which map to JavaScript ``number`` values
```

**Pull request merged**